### ISSUE TITLE
Old Cover Blocks with Fixed Backgrounds are now showing greatly enlarged backgrounds.

### ISSUE THREAD

**Issue by KirkwallDay** [Automattic/wp-calypso#65042]
### Quick summary

This is one I cannot reproduce, but I can give examples. It seems cover blocks with older code that worked well in January (could have been later) are now enlarging the image greatly.

Testing this on a test site is a problem as currently this issue is a blocker to testing: https://github.com/Automattic/wp-calypso/issues/64682 setting a new cover block results in the repeated background being added even if it's not selected. So we can't create a new cover block to see if the size is enlarged abnormally.

You can find an example in this ticket: 5329121-zen
And as this involves customer data, details are in this p2 post: pd4QKa-Bt-p2

### Steps to reproduce

See the p2 post for specific details, this involves customer data and is current not possible to reproduce due to the other Github issue.

### What you expected to happen

The cover blocks should not enlarge the pictures to huge dimensions for a fixed background. 

### What actually happened

They should look as they did in January.

### Context

5329121-zen

### Simple, Atomic or both?

Simple, Atomic

### Reproducibility

Intermittent

### Severity

Some (< 50%)

### Available workarounds?

No but the platform is still usable

### Workaround details

Setting the background to fixed will resolve the enlargement issue and repeated background issue.

**Comment by KirkwallDay** [comment]
Another instance here: 5329926-zen

Kindly follow up here: 

- [ ] 5329926-zd-woothemes

**Comment by happychait** [comment]
This customer was promised to be updated when this bug is fixed.

- [ ]  5329121-zen

**Comment by filipanoscampos** [comment]
Another case here 35947281-hc

Kindly follow up here:

- [ ] 5339064-zd-woothemes

**Comment by KirkwallDay** [comment]
Another instance here: 5325449-zen

Kindly Follow-up here:

- [ ] 5325449-zen

**Comment by AMcGrady** [comment]
Another instance here: 5339064-zen

Kindly Follow-up here:

- [ ] 5339064-zen

**Comment by essleeung** [comment]
Another instance here: 5342352-zd-woothemes
I've asked them to not set their cover block to fixed for now.

**Comment by dcoleonline** [comment]
- [ ] 5342352-zd

**Comment by msilbers** [comment]
- [ ] 5341159-zd-woothemes

**Comment by cuemarie** [comment]
📌 SCRUBBING
- Tested on P2 ✅ 
- Tested on Simple ✅ 
- Tested on AT ✅ 
- Tested on Self-hosted ✅ 
- Replicable on Core - **Yes**

📌 FINDINGS/SCREENSHOTS/VIDEO
- I was not blocked by https://github.com/Automattic/wp-calypso/issues/64682 in testing this
- I found that any new cover block added to any test site also zoomed in on the image when `Fixed background` was enabled. However, it's unclear at the moment if this is a bug, or perhaps a new intended behavior.

**For tests, I started by copying over cover blocks from the OP's site**. Cover blocks zoomed in on the image in all testing environments, as well as on new cover blocks added today.

**AT**

https://user-images.githubusercontent.com/27249804/178546697-367147c7-4cef-4aed-acbc-653cf16ba84e.mp4

**Self-Hosted**

https://user-images.githubusercontent.com/27249804/178547047-aec2fcc1-59a1-4638-a62f-a633e5fa2671.mp4

**Simple P2**

https://user-images.githubusercontent.com/27249804/178547076-0f14bbb2-a368-4633-b98e-caf58d795414.mp4

**Image Dimensions**
It seems that how far the block zooms in is determined by the original file size. For instance, the user image in these tests has huge dimensions: 

<img width="1074" alt="Markup 2022-07-12 at 09 37 51" src="https://user-images.githubusercontent.com/27249804/178546898-c121e116-d78e-430d-8b51-03204b45f93a.png">

📌 **Workaround**
- As a workaround, users impacted by this can resize the background image down to impact how much the image zooms in. For example: 

I resized the OP user's `5830 x 3887` image down to `1458 x 972`, and here's how the block behaves now: 

![Markup on 2022-07-12 at 09:46:30](https://user-images.githubusercontent.com/27249804/178548322-f9054ad7-abf0-41be-8b35-4b21e3e12534.png)

https://user-images.githubusercontent.com/27249804/178548496-894769ab-e32e-47d3-96e8-d60b248caa71.mov

---

📌 ACTIONS
- Marked as Triaged for Quality Squad review
- I'll raise this with the PAs to see if we can get some folks to weigh in on whether this behavior is by design

**Comment by annezazu** [comment]
Hi hi! I don't have access to the various zendesk tickets and am not sure what p2 is being referenced since nothing is linked. The Cover block had some style changes in 5.9 (wouldn't impact this) and can now use a featured image in 6.0 (again wouldn't impact this afaik) so I'm not quite sure what's different here. 

I did try various sized images using WordPress 6.0.1 and wasn't able to notice anything super odd:

https://user-images.githubusercontent.com/26996883/178573413-301aab9b-cd3c-4ca4-ab2b-b92cc0626f56.mov

When I try with WordPress 6.0.1 and Gutenberg 13.6, I do notice a few oddities that I think make sense to report in the Gutenberg repo:

https://user-images.githubusercontent.com/26996883/178573647-a8c3b602-bc14-4e74-8ef9-f8cbca7c3bcd.mov

Specifically, the largest image size zooms in more whereas the smallest ends up as a pattern! @cuemarie can you handle following up on this please?

**Comment by msilbers** [comment]
- [ ] 5373274-zd-woothemes

The customers coming in are specifically saying that it looks different than previously. It used to be much less zoomed in.

**Comment by annezazu** [comment]
Definitely think an issue in Gutenberg's GitHub repo needs to be opened with examples, clear ways to reproduce, etc.

**Comment by supernovia** [comment]
> clear ways to reproduce

@annezazu If it's only an issue happening with previously-created old cover blocks, would it make sense to just grab the affected code and work with that?

**Comment by annezazu** [comment]
Yes! That would be very helpful to cross compare to see what might be at the root of this. Specifically, examples along with timeframes of when this changed would go a long way.

**Comment by annezazu** [comment]
To be safe/to not lose more time, I went ahead and opened an issue in the Gutenberg repo: https://github.com/WordPress/gutenberg/issues/42407 As a reminder, I cannot see the zendesk tickets so if folks can share examples of the block markup impacted by this issue in the Gutenberg one, that would be very helpful!

**Comment by TPilgs** [comment]
Another instance here: 34135037-hc

User requested to be notified if/when it's fixed.

**Comment by mxhassani** [comment]
Another one: 5401674-zd-woothemes

**Comment by cuemarie** [comment]
@annezazu Coming back to this after some time away from guild work, thank you for pushing forward with the Gutenberg issue!  Is this the kind of code that's helpful?

![Markup on 2022-07-17 at 13:45:03](https://user-images.githubusercontent.com/27249804/179424148-66e4a0e4-62b0-4803-9725-413fc63205fc.png)

Here's the full copiable code from the OP's cover block, I've just swapped the Image URLs with a free Pexels photo:

```
<!-- wp:cover {"url":"https://images.pexels.com/photos/2480072/pexels-photo-2480072.jpeg","id":333,"dimRatio":20,"customOverlayColor":"#000000","contentPosition":"center center","isDark":false,"align":"full","className":"is-style-default"} -->
<div class="wp-block-cover alignfull is-light is-style-default"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-20 has-background-dim" style="background-color:#000000"></span><img class="wp-block-cover__image-background wp-image-333" alt="" src="https://images.pexels.com/photos/2480072/pexels-photo-2480072.jpeg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:spacer {"height":"20px"} -->
<div style="height:20px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:separator {"opacity":"css","backgroundColor":"background","className":"is-style-wide"} -->
<hr class="wp-block-separator has-text-color has-background-color has-css-opacity has-background-background-color has-background is-style-wide"/>
<!-- /wp:separator -->

<!-- wp:heading {"textAlign":"center","style":{"typography":{"fontWeight":"200","fontStyle":"normal","textTransform":"lowercase","fontSize":"35px"}},"textColor":"secondary","className":"alignwide"} -->
<h2 class="has-text-align-center alignwide has-secondary-color has-text-color" id="un-chemin-vers-le-bien-etre" style="font-size:35px;font-style:normal;font-weight:200;text-transform:lowercase"><mark style="background-color:rgba(0,0,0,0);" class="has-inline-color has-white-color">Un chemin vers le bien être ... </mark></h2>
<!-- /wp:heading -->

<!-- wp:separator {"opacity":"css","backgroundColor":"background","className":"is-style-wide"} -->
<hr class="wp-block-separator has-text-color has-background-color has-css-opacity has-background-background-color has-background is-style-wide"/>
<!-- /wp:separator -->

<!-- wp:spacer {"height":"20px"} -->
<div style="height:20px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:paragraph -->
<p></p>
<!-- /wp:paragraph --></div></div>
<!-- /wp:cover -->
```

I'll share on the Gutenberg issue as well.

**Comment by cuemarie** [comment]
**Update for HEs** - A fix is in the works here: https://github.com/WordPress/gutenberg/pull/42470 . Gutenberg updates are released every 2 weeks (rolling out on WordPress.com sites shortly after that), so once this fix is confirmed (not quite there yet as of this comment), it'll be another two weeks or so for the solution to take effect on WordPress.com sites.

In the meantime, if you'd like to offer a CSS workaround,  you can address the issue on individual sites frontend display by adding the following custom CSS (Thanks @glendaviesnz on https://github.com/WordPress/gutenberg/issues/42407 for this!):

```
/* Cover Block Fixed BG - CSS workaround | XXX-hc/zen [XX] */
.wp-block-cover-image.has-parallax,
.wp-block-cover.has-parallax,
.wp-block-cover__image-background.has-parallax,
video.wp-block-cover__video-background.has-parallax {
	background-size: cover;
	background-repeat: no-repeat;
}
```

**Comment by annezazu** [comment]
@cuemarie Thanks for sharing! That is helpful. No more examples needed at this point since a fix is in the works.

**Comment by cuemarie** [comment]
Copy that, @annezazu ! 

For HEs that encounter users who want an update when this one is fixed, we've queued this up for **batch replies** when the time comes, so feel free to share any other tickets that need a follow up. Formatting reminders: p7DVsv-eD3-p2

**Comment by syedshaon** [comment]
Another one: 5382860-zd-woothemes

**Comment by arunsathiya** [comment]
@alliebbush2 shared a CSS in 5408423-zen. Tracking here in case we want to followup once the bug is resolved and CSS removed.

**Comment by daledupreez** [comment]
Noting that WordPress/gutenberg#42470 addresses the issue and was merged yesterday. The fix should be included in Gutenberg 13.8.

**Comment by jp-imagines** [comment]
- [ ] Another report in 5430986-zd-woothemes.

(Personal plan, so no CSS. Suggested disabling the "fixed background" setting for now, or shrinking the affected images as noted above, as temporary workarounds while we wait for the fix to be merged.)

**Comment by christianguerrag** [comment]
The user is reaching out for an update here 6433029-zen, it is said that the fix was to be deployed in Gutenberg 13.8, is there an update on the issue?

**Comment by cuemarie** [comment]
@christianguerrag It looks like this was resolved, yes!  The issue just wasn't closed when https://github.com/WordPress/gutenberg/pull/42470 was merged.  

I took a quick look at the user in your ticket, and all their media images are HUUUUGE - like upwards of 3000-4000px in dimensions, and 4-8MB.  It's possible they're just seeing the fixed cover block zoom in because of the size of their images, and a down-sized image should work better.

I hope this helps! Thanks for the ping on this.

📌 ACTIONS
- Closing this now as resolved. Will look into batch replies for impacted users soon.

📌 Message to HEs
- Since Gutenberg has reached v16.0.0 as of this post, and this issue was resolved with Gutenberg 13.8, please open a new issue for new reports of cover blocks with fixed backgrounds that appear to be broken (feel free to refer to this if you feel it's similar). This will help with clarity if there's a new underlying cause for an issue.
